Sources/BranchSDK/BNCServerRequestQueue.m

```diff
@@ -279,6 +281,11 @@ - (void)retrieve {
         NSData *data = [NSData dataWithContentsOfURL:self.class.URLForQueueFile options:0 error:&error];
         if (!error && data) {
             NSMutableArray *decodedQueue = [self unarchiveQueueFromData:data];
+            for (id request in decodedQueue) {
+                if ([request isKindOfClass:[BranchOpenRequest class]] || [request isKindOfClass:[BranchInstallRequest class]]) {
+                    ((BranchOpenRequest *)request).isFromArchivedQueue = YES;
```

Comment:
Does this type cast work if it's BranchInstallRequest?

Reply:
Yes.  BranchOpenRequest is superclass of BranchInstallRequest.